provider/repodata/data/json/segmenter/lstm@1/th.json

```diff
@@ -1,5 +1,5 @@
 {
-  "model": "Thai_codepoints_exclusive_model4_heavy",
```

Comment:
Nit (optional): I do think it's useful to retain memory of the whole model name, perhaps alongside the codepoint/grapheme enum.